.changeset/tall-peas-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@powersync/service-sync-rules': minor
```

Comment:
I've set this to minor because breaking bumps it to 1.0.0 - there are breaking changes.